bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -35,7 +36,7 @@ deploy_rancher() {
         --install rancher rancher-latest/rancher \
         --version "${RD_RANCHER_IMAGE_TAG#v}" \
         --namespace cattle-system \
-        --set hostname="$(get_host)" \
+        --set hostname="$(get_host || return)" \
```

Comment:
I don't like the `return` nested within parameter substitution inside a big block of parameters. If this is necessary, then I would prefer something like:

```bash
local host
host=$(get_host) || return
helm upgrade … \\
    --set hostname="$host"
```

We really need to figure out how to enable `errexit` inside `run`, to get rid of this silliness.

Reply:
Changed. (Also accidentally pushed this branch to upstream; deleted it.)